packages/webhooks/lib/sync.ts

```diff
@@ -58,6 +58,7 @@ export const sendSync = async ({
     };
 
     if (success) {
+        body.queryTimeStamp = null;
```

Comment:
```suggestion
        body.queryTimeStamp = syncType !== 'INITIAL' ? now as unknown as string : null;
```
and remove the if below

Reply:
Updated